### PR TITLE
#163165226 Add jwt Token And Refactor Test to have Auth Header

### PR DESCRIPTION
**Pull request by Ogollah**
### What does this PR do?
Adds jwt auth token  for  QUESTIONER API

### Description of Task to be completed?
Have these endpoints working
```
|Endpoint                            | Functionality                    |HTTP method 
|------------------------------------|----------------------------------|-------------
|/api/v1/user/signup                 |Signup account                    |POST        
|/api/v1/user/auth/signin            |Signin in user                    |POST
|/api/v1/user/auth/signout           |Signout in user                   |POST
|/api/v1/meetups/create              |Create a meetup record            |POST
|/api/v1/meetups/upcoming            |Get upcoming meetups              |GET
|/api/v1/meetups/<meetup-id>         |Fetch specific upcoming meetup    |GET
|/api/v1/questions/questions         |Fetch all question                |GET
|/api/v1/questions/<meetup-id>/create|Create a question                 |GET
```

### How should this be manually tested?
Using your terminal run the following commands:
```
- cd path/to/server-root
- git clone this repo
- cd to Questioner
- python3 -m venv env
- source env/bin/activate
- pip install requirement.txt
- on command line run manage.py test or run
```
### Any background context you want to provide?
No database
### What are the relevant pivotal tracker stories?
[#163165226](https://www.pivotaltracker.com/story/show/163165226)

### CODE REVIEW

**Comment by matthenge** [comment]
Awesome. Great work

File: api/v1/main/model/meepup.py [hunk 1]
```diff
@@ -24,7 +24,7 @@ def __repr__(self):
```
Comment (oagutu) [inline api/v1/main/model/meepup.py:24]: Rename this file to `meetup.py` instead of `meepup.py`

**Review by oagutu — COMMENTED** [review]
Good work on this. The app crashes when either an invalid token or no token provided

File: api/v1/main/model/meepup.py (hunk 1 above)
Comment (Ogollah) [reply api/v1/main/model/meepup.py:24]: Noted

**Pull request merged**